### ISSUE TITLE
Docker 2.1.0.1 for Windows unusably slow

### ISSUE THREAD

**Issue by yoyo-array0825** [docker/for-win#4476]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 704140D2-93CE-47D6-8533-2A67B7D774C9/20190808152205

### Expected behavior
Docker CLI (cmd) commands return output

### Actual behavior
Docker CLI commands run continuously without ever failing or returning anything. This includes commands like 'docker --help' and 'docker info'.

### Information

  - Windows Version:
  - Docker Desktop Version:

### Steps to reproduce the behavior

How I got here
  1. Updated Docker to latest
  2. ...

**Comment by mikeparker** [comment]
I can't see any errors in your log file, and the docker engine is booting reasonably quickly.

One thing I noticed was from your settings file:
`"dataFolder": "C:\\Users\\Public\\Documents\\Hyper-V\\Virtual Hard Disks"`

Is there a reason your computer has the Hyper-V disk location in a publicly shared folder?

Also have you tried switching to windows container mode?

**Comment by sgammill** [comment]
"C:\\Users\\Public\\Documents\\Hyper-V\\Virtual Hard Disks" is Hyper-V's default location for storing virtual disks.  ACLs prevent access to non-admins.

**Comment by yoyo-array0825** [comment]
@mikeparker I never got an error -- Docker just sort of stalled and sat there trying, so no error message makes sense (thanks for confirming that). ... I ended up reinstalling (which didn't help) and then restoring factory settings, and that did the trick.

I found some weird goings-on, and I think some containers weren't getting removed as expected and may have hijacked the ports I was trying to use. Did not find any other weirdness ...
